zipline/finance/trading.py

```diff
@@ -149,44 +157,67 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         # stack.
         return False
 
-    def update_asset_finder(self,
-                            clear_metadata=False,
-                            asset_finder=None,
-                            asset_metadata=None,
-                            identifiers=None):
+    def write_data(self,
```

Comment:
Should this method be named something like 'write_asset_metadata'? TradingEnvironment.write_data seems really vague to me - from the outside, I wouldn't know what data was being written.